### DISCUSSION TITLE
Using Multiple Tables As for Formula For Pricing Matrix

### DISCUSSION THREAD

**Discussion by lardofpatato** [nocodb/nocodb#1580] [General]
Hi, I am having trouble creating a pricing matrix due to not being able to multiple (Use Data from) columns from another table I have got from using the lookup function. is their alt way to get data from another table I'm not aware of to use to make a formula from.
![nocodb problem](https://user-images.githubusercontent.com/101444506/161357269-e3e7fef5-2dc1-4be3-8f14-89946731667f.png)

**Comment by o1lab** [comment]
This is coming next week (stay tuned).
